pkg/kubectl/cmd/util/cmdproto/k8s_io_kubectl_cmd/cmd.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto2";
+package k8s.io.kubectl.cmd;
+
+import "google/protobuf/descriptor.proto";
+
+message FlagDetail {
+  required string name = 1;
+  repeated string value = 2;
+  required string shorthand = 3;
+  required string usage = 4;
+  enum EXT {
+      NONE = 0;
+      ISTIME = 1;
+      ISSTRARRAY = 2;
+      ISSTRSLICE = 3;
+  }
+  optional EXT ext = 5;
+}
+
+extend google.protobuf.FieldOptions {
+  optional FlagDetail info = 1234;
+}
+
+message Array {
+    repeated string array = 1;
+}
+
+//Flags Description
+
+
+//This is the flag description for `kubectl version`
+message VersionFlags {
+  optional bool client = 2 [ (info) = { name: "client" value: "false" shorthand: "c" usage: "Client version only (no server required)."}];
+  optional bool short = 3  [ (info) = { name: "short" value: "false" shorthand: "" usage: "Print just the version number."}];
+  optional string output = 4 [ (info) = { name: "output" value: "" shorthand: "o" usage: "One of 'yaml' or 'json'."}];
+}
+
+//This is the flag description only for test
+message TestTest {
```

Comment:
WDYT of adding support for parsing the command description and example as options as well.